disco_aws_automation/disco_bake.py

```diff
@@ -445,12 +445,12 @@ def bake_ami(self, hostclass, no_destroy, source_ami_id=None, stage=None):
         return image
 
     @staticmethod
-    def _tag_ami_with_metadata(ami, stage, source_ami_id, productline=None):
+    def _tag_ami_with_metadata(ami, stage, source_ami_id, productline=None, extra_tags=None):
         """
         Tags an AMI with the stage, source_ami, the branch/git-hash of disco_aws_automation,
         and the productline if provided
         """
-        tag_dict = OrderedDict()
+        tag_dict = extra_tags or OrderedDict()
```

Comment:
If `extra_tags` exists, will then `tag_dict` be of type OrderedDict? Why do we need it to be OrderedDict?